### PR TITLE
Increased efficiency and speed of webpack compilation

### PR DESCRIPTION
**Pull request by GilianJuice**
This is done by splitting vendor scripts into separate chunks, so that they are no longer included in each module. This also makes loading scripts in modules faster and more efficient.

Asana ticket: https://app.asana.com/0/1201027711166952/1203325592628572

### CODE REVIEW

File: FrontEnd/Core/Scripts/main.js [hunk 1]
```diff
@@ -51,910 +51,911 @@ import {
...
-            if (this.appSettings.trackJsToken) {
-                try {
-                    TrackJS.install({
-                        token: this.appSettings.trackJsToken
-                    });
-                } catch(exception) {
-                    console.error("Error loading TrackJS widget", exception);
-                }
-            }
+console.log("main.js loaded");
```
Comment (MarkVanDijkHG) [inline FrontEnd/Core/Scripts/main.js:54]: Console log mag weg.

File: FrontEnd/Core/Scripts/main.js [hunk 2]
```diff
@@ -51,910 +51,911 @@ import {
...
-                        generalMessagePromptText: "",
-                        createBranchSettings: {
-                            name: null,
-                            startMode: "direct",
-                            startOn: null,
-                            entities: {
-                                all: {
-                                    mode: 0
-                                }
+        switch (event.data.action) {
```
Comment (MarkVanDijkHG) [inline FrontEnd/Core/Scripts/main.js:132]: Aangezien er maar 1 case in de switch statement zit, kan dit niet beter een if statement worden?

File: FrontEnd/Core/Scripts/main.js [hunk 3]
```diff
@@ -51,910 +51,911 @@ import {
...
-            this.vueApp = this.vueApp.mount("#app");
-        }
+        // Mount our app to the main HTML element.
+        this.vueApp = this.vueApp.mount("#app");
     }
+}
 
-    window.main = new Main();
-})();
+console.log("initiating main class");
```
Comment (MarkVanDijkHG) [inline FrontEnd/Core/Scripts/main.js:960]: Console log mag weg.

**Review by MarkVanDijkHG — CHANGES_REQUESTED** [review]
Paar kleine dingen even controleren.

File: FrontEnd/Core/Scripts/main.js (hunk 1 above)
Comment (GilianJuice) [reply FrontEnd/Core/Scripts/main.js]: Weggehaald.

File: FrontEnd/Core/Scripts/main.js (hunk 2 above)
Comment (GilianJuice) [reply FrontEnd/Core/Scripts/main.js:130]: Weggehaald.

File: FrontEnd/Core/Scripts/main.js (hunk 3 above)
Comment (GilianJuice) [reply FrontEnd/Core/Scripts/main.js]: Weggehaald.

**Review by MarkVanDijkHG — APPROVED** [review]

**Pull request merged**